taxonomy/providers/__init__.py

```diff
@@ -4,3 +4,4 @@
 """
 
 from taxonomy.providers.course_metadata import CourseMetadataProvider
+from taxonomy.providers.program import ProgramProvider
```

Comment:
a thought: i think we should be coherent with the naming pattern here e.g. `CourseMetadataProvider` and `ProgramMetadataProvider`

Reply:
+1